ui/accordion.js

```diff
@@ -499,7 +502,7 @@ return $.widget( "ui.accordion", {
 			});
 		} else if ( toShow.length ) {
 			this.headers.filter(function() {
-				return $( this ).attr( "tabIndex" ) === 0;
+				return $( this ).attr( "tabIndex" ) === "0";
```

Comment:
You probably want to use `parseInt` for the best compat. I think on older versions core returns a number when it should return a string. Looks like it changed in 1.8.something  http://jsfiddle.net/qvpq4wo1/